test/nodes/test_connector.py

```diff
@@ -142,3 +146,18 @@ def test_crawler_return_document(test_url, tmp_path):
             file_content = json.load(doc_file)
             assert file_content["meta"] == document.meta
             assert file_content["content"] == document.content
+
+
+def test_crawler_extract_hidden_text(test_url, tmp_path):
+    crawler = Crawler(output_dir=tmp_path)
+    documents, _ = crawler.run(
+        urls=[test_url + "/page_w_hidden_text.html"], extract_hidden_text=True, crawler_depth=0, return_documents=True
+    )
+    crawled_content = documents["documents"][0].content
+    assert "hidden text" in crawled_content
+
+    documents, _ = crawler.run(
+        urls=[test_url + "/page_w_hidden_text.html"], extract_hidden_text=False, crawler_depth=0, return_documents=True
+    )
+    crawled_content = documents["documents"][0].content
+    assert "hidden text" not in crawled_content
```

Comment:
I think this part can be split out in a separate test. I like tests with one assert only :smile: